tracer/src/Datadog.Trace/Ci/Coverage/Attributes/CoveredAssemblyAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="CoveredAssemblyAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Ci.Coverage.Attributes
+{
+    /// <summary>
+    /// Covered assembly attribute
+    /// This attributes marks an assembly as a processed.
+    /// </summary>
+    public class CoveredAssemblyAttribute : Attribute
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CoveredAssemblyAttribute"/> class.
+        /// </summary>
+        /// <param name="totalMethods">Total number of methods</param>
+        /// <param name="totalInstructions">Total number of instructions</param>
+        public CoveredAssemblyAttribute(ulong totalMethods, ulong totalInstructions)
+        {
+            TotalMethods = totalMethods;
+            TotalInstructions = totalInstructions;
+        }
+
+        /// <summary>
+        /// Gets total number methods
+        /// </summary>
+        public ulong TotalMethods { get; }
```

Comment:
How are these two properties being used? I may have missed it, but I only saw these values passed into the constructor but not accessed later

Reply:
Yeah, is not used by the code, I will remove it.

Reply:
done in [d0d8115](https://github.com/DataDog/dd-trace-dotnet/pull/2739/commits/d0d8115612e93d65e6a09402f69f3443e1a35bfb)